python/cuspatial/cuspatial/tests/test_lonlat2coord.py

```diff
@@ -1,98 +1,98 @@
 # Copyright (c) 2019, NVIDIA CORPORATION.
 
 import pytest
+
 import cudf
 from cudf.tests.utils import assert_eq
-import numpy as np
+
 import cuspatial
 
+
 def test_camera_oob_0():
     with pytest.raises(RuntimeError):
-        result = cuspatial.lonlat_to_xy_km_coordinates(-200, 0,
-            cudf.Series([0]),
-            cudf.Series([0])
+        result = cuspatial.lonlat_to_xy_km_coordinates(  # noqa: F841
+            -200, 0, cudf.Series([0]), cudf.Series([0])
         )
 
+
 def test_camera_oob_1():
     with pytest.raises(RuntimeError):
-        result = cuspatial.lonlat_to_xy_km_coordinates(200, 0,
-            cudf.Series([0]),
-            cudf.Series([0])
+        result = cuspatial.lonlat_to_xy_km_coordinates(  # noqa: F841
+            200, 0, cudf.Series([0]), cudf.Series([0])
         )
 
+
 def test_camera_oob_2():
     with pytest.raises(RuntimeError):
-        result = cuspatial.lonlat_to_xy_km_coordinates(0, -100,
-            cudf.Series([0]),
-            cudf.Series([0])
+        result = cuspatial.lonlat_to_xy_km_coordinates(  # noqa: F841
+            0, -100, cudf.Series([0]), cudf.Series([0])
         )
 
+
 def test_camera_oob_3():
     with pytest.raises(RuntimeError):
-        result = cuspatial.lonlat_to_xy_km_coordinates(0, 100,
-            cudf.Series([0]),
-            cudf.Series([0])
+        result = cuspatial.lonlat_to_xy_km_coordinates(  # noqa: F841
+            0, 100, cudf.Series([0]), cudf.Series([0])
         )
 
+
 @pytest.mark.parametrize("corner", [0, 1, 2, 3])
 def test_camera_corners(corner):
     x = [-180, 180, -180, 180]
     y = [-90, 90, 90, -90]
-    result = cuspatial.lonlat_to_xy_km_coordinates(x[corner], y[corner],
-        cudf.Series(x[corner]),
-        cudf.Series(y[corner])
+    result = cuspatial.lonlat_to_xy_km_coordinates(
+        x[corner], y[corner], cudf.Series(x[corner]), cudf.Series(y[corner])
     )
-    result = cudf.DataFrame({'x': [0], 'y': [0]})
+    assert_eq(result, cudf.DataFrame({"x": [0], "y": [0]}))
```

Comment:
Not sure if this was what the test was meant to do.

@thomcom?

Reply:
Nice catch, thanks.

Reply:
Had to make them floats, but test passes for me now, woohoo!